pkg/rpcclient/wsclient.go

```diff
@@ -808,11 +808,13 @@ func (c *WSClient) ReceiveExecutions(flt *neorpc.ExecutionFilter, rcvr chan<- *s
 }
 
 // ReceiveNotaryRequests registers provided channel as a receiver for notary request
-// payload addition or removal events. Events can be filtered by the given TxFilter
+// payload addition or removal events. Events can be filtered by the given NotaryRequestFilter
 // where sender corresponds to notary request sender (the second fallback transaction
-// signer) and signer corresponds to main transaction signers. nil value doesn't add
-// any filter. See WSClient comments for generic Receive* behaviour details.
-func (c *WSClient) ReceiveNotaryRequests(flt *neorpc.TxFilter, rcvr chan<- *result.NotaryRequestEvent) (string, error) {
+// signer) and signer corresponds to main transaction signers, type corresponds to the
```

Comment:
s/`signer) and signer`/`signer), signer`
s/`signers, type corresponds`/`signers and type corresponds`